<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>1096</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QFrame {
	background: rgb(186, 189, 182);
    border-style: solid;
    border-color: rgb(255, 255, 255);
    border-width: 1px;
    border-radius: 6px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    color: rgb(0, 0, 0);
	font: 16pt &quot;Bebas Kai&quot;;
}</string>
    </property>
    <property name="text">
     <string>Points on a circle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_47">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>470</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
    border-style: solid;
    border-color: black;
    border-width: 2px;
    border-radius: 6px;
    color: rgb(238, 238, 236);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(81, 86, 85), stop:0.489795 rgb(99, 102, 102), stop:0.699799 rgb(85, 88, 94), stop:0.90444 rgb(77, 84, 86), stop:0.160246 rgb(83, 84, 91), stop:1 rgb(109, 115, 118));
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_201">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_142">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 75 13pt &quot;Bebas Kai&quot;;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>CENTER Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FloatLineEdit" name="center_y_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>70</horstretch>
         <verstretch>33</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Bebas Kai&quot;;</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="bottom" stdset="0">
        <double>-179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_42">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
    border-style: solid;
    border-color: black;
    border-width: 2px;
    border-radius: 6px;
    color: rgb(238, 238, 236);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(81, 86, 85), stop:0.489795 rgb(99, 102, 102), stop:0.699799 rgb(85, 88, 94), stop:0.90444 rgb(77, 84, 86), stop:0.160246 rgb(83, 84, 91), stop:1 rgb(109, 115, 118));
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_195">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_140">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 75 13pt &quot;Bebas Kai&quot;;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>START
ANGLE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FloatLineEdit" name="start_angle_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>70</horstretch>
         <verstretch>33</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Bebas Kai&quot;;</string>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="format_string" stdset="0">
        <string>%.2f</string>
       </property>
       <property name="bottom" stdset="0">
        <double>-179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_25">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
    border-style: solid;
    border-color: black;
    border-width: 2px;
    border-radius: 6px;
    color: rgb(238, 238, 236);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(81, 86, 85), stop:0.489795 rgb(99, 102, 102), stop:0.699799 rgb(85, 88, 94), stop:0.90444 rgb(77, 84, 86), stop:0.160246 rgb(83, 84, 91), stop:1 rgb(109, 115, 118));
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_193">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_139">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 75 13pt &quot;Bebas Kai&quot;;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Num
Holes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="IntLineEdit" name="num_holes_input">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Bebas Kai&quot;;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="default_value" stdset="0">
        <number>0</number>
       </property>
       <property name="bottom" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_46">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>415</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
    border-style: solid;
    border-color: black;
    border-width: 2px;
    border-radius: 6px;
    color: rgb(238, 238, 236);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(81, 86, 85), stop:0.489795 rgb(99, 102, 102), stop:0.699799 rgb(85, 88, 94), stop:0.90444 rgb(77, 84, 86), stop:0.160246 rgb(83, 84, 91), stop:1 rgb(109, 115, 118));
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_199">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_141">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 75 13pt &quot;Bebas Kai&quot;;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>CENTER X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FloatLineEdit" name="center_x_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>70</horstretch>
         <verstretch>33</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Bebas Kai&quot;;</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="bottom" stdset="0">
        <double>-179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>145</y>
      <width>481</width>
      <height>366</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../../resources.qrc">:/images/holes_circle.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    color: rgb(0, 0, 0);
	font: 18pt &quot;Bebas Kai&quot;;
}</string>
    </property>
    <property name="text">
     <string>Define points where to make operations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_selected_input">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>480</y>
      <width>181</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>42</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>42</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bebas Kai&quot;;</string>
    </property>
    <property name="text">
     <string>DELETE SELECTED</string>
    </property>
   </widget>
   <widget class="QTableView" name="xy_coord_input">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>125</y>
      <width>416</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView,
QHeaderView {
    font: 14pt &quot;Bebas Kai&quot;;
	background-color: rgb(220, 220, 220);
	color: black;
    border: none;
}

QTableView {
   	border-top: 8px rgb(120, 120, 120);
	border-left: 4px  rgb(120, 120, 120);
	border-bottom: 5px rgb(120, 120, 120);
	border-right: 4px rgb(120, 120, 120);
	border-radius: 5px;
	border-color: rgb(120, 120, 120);
	border-style: solid;
	background-color: rgb(120, 120, 120);
    gridline-color: rgb(203, 203, 203);
	alternate-background-color: rgb(90, 90, 90);
}</string>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    color: rgb(0, 0, 0);
	font: 16pt &quot;Bebas Kai&quot;;
}</string>
    </property>
    <property name="text">
     <string>Specified XY Points</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_all_input">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>166</width>
      <height>42</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>42</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>42</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bebas Kai&quot;;</string>
    </property>
    <property name="text">
     <string>DELETE ALL</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_55">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>310</y>
      <width>140</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
    border-style: solid;
    border-color: black;
    border-width: 2px;
    border-radius: 6px;
    color: rgb(238, 238, 236);
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(81, 86, 85), stop:0.489795 rgb(99, 102, 102), stop:0.699799 rgb(85, 88, 94), stop:0.90444 rgb(77, 84, 86), stop:0.160246 rgb(83, 84, 91), stop:1 rgb(109, 115, 118));
}</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_215">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_149">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>33</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font: 75 13pt &quot;Bebas Kai&quot;;
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Diam</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FloatLineEdit" name="diameter_input">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>70</horstretch>
         <verstretch>33</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>33</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bebas Kai</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
    border-style: transparent;
    border-color: rgb(235, 235, 235);
    border-width: 1px;
    border-radius: 5px;
    color: black;
    background: white;
	font: 14pt &quot;Bebas Kai&quot;;
}</string>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="format_string" stdset="0">
        <string>{0:.3f}</string>
       </property>
       <property name="bottom" stdset="0">
        <double>-179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnDoneEditing">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>610</y>
      <width>206</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Done Defining Points</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>745</x>
      <y>610</y>
      <width>83</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>frame_47</zorder>
   <zorder>frame_42</zorder>
   <zorder>frame_25</zorder>
   <zorder>frame_46</zorder>
   <zorder>label_4</zorder>
   <zorder>delete_selected_input</zorder>
   <zorder>xy_coord_input</zorder>
   <zorder>label_3</zorder>
   <zorder>delete_all_input</zorder>
   <zorder>frame_55</zorder>
   <zorder>btnDoneEditing</zorder>
   <zorder>btnCancel</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IntLineEdit</class>
   <extends>VCPLineEdit</extends>
   <header>widgets.conversational.int_line_edit</header>
  </customwidget>
  <customwidget>
   <class>FloatLineEdit</class>
   <extends>VCPLineEdit</extends>
   <header>widgets.conversational.float_line_edit</header>
  </customwidget>
  <customwidget>
   <class>VCPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qtpyvcp.widgets.input_widgets.line_edit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
